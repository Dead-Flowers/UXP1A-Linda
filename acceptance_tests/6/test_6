-t 5 output '(1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1)'
-t 5 output '(\"dsagfhgshgfugfuhhfgushfhgaufhghjshvjabdhhshjdvgjusfhjhaiwfhsbdivjshfihshjfhjabfjshjvjgbauhfjsbdivhhjshdfijhsfdjalfopjbakjkhkjakkjamsdnfhsdhfojahjfbdkfjgjahfkijsjhfkajgfgjhdsagfhgshgfugfuhhfgushfhgaufhghjshvjabdhhshjdvgjusfhjhaiwfhsbdivjshfihshjfhjabfjshjvjgbauhfjsbdivhhjshdfijhsfdjalfopjbakjkhkjakkjamsdnfhsdhfojahjfbdkfjgjahfkijsjhfkajgfgjhdsagfhgshgfugfuhhfgushfhgaufhghjshvjabdhhshjdvgjusfhjhaiwfhsbdivjshfihshjfhjabfjshjvjgbauhfjsbdivhhjshdfijhsfdjalfopjbakjkhkjakkjamsdnfhsdhfojahjfbdkfjgjahfkijsjhfkajgfgjhdsagfhgshgfugfuhhfgushfhgaufhghjshvjabdhhshjdvgjusfhjhaiwfhsbdivjshfihshjfhjabfjshjvjgbauhfjsbdivhhjshdfijhsfdjalfopjbakjkhkjakkjamsdnfhsdhfojahjfbdkfjgjahfkijsjhfkajgfgjhdsagfhgshgfugfuhhfgushfhgaufhghjshvjabdhhshjdvgjusfhjhaiwfhsbdivjshfihshjfhjabfjshjvjgbauhfjsbdivhhjshdfijhsfdjalfopjbakjkhkjakkjamsdnfhsdhfojahjfbdkfjgjahfkijsjhfkajgfgjhdsagfhgshgfugfuhhfgushfhgaufhghjshvjabdhhshjdvgjusfhjhaiwfhsbdivjshfihshjfhjabfjshjvjgbauhfjsbdivhhjshdfijhsfdjalfopjbakjkhkjakkjamsdnfhsdhfojahjfbdkfjgjahfkijsjhfkajgfgjhdsagfhgshgfugfuhhfgushfhgaufhghjshvjabdhhshjdvgjusfhjhaiwfhsbdivjshfihshjfhjabfjshjvjgbauhfjsbdivhhjshdfijhsfdjalfopjbakjkhkjakkjamsdnfhsdhfojahjfbdkfjgjahfkijsjhfkajgfgjh\")'
