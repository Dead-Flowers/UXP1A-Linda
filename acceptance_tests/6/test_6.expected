Exit code: 1
Exit code: 1
